src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -6,6 +6,7 @@
 import java.util.Collections;
 import java.util.Iterator;
 import java.util.List;
+import java.util.*;
```

Comment:
This currently violates the java coding standards: **Imported classes should always be listed explicitly.**

You can prevent IntelliJ from doing so by going through,
`Settings/Preferences -> Code Style -> Java -> Check "Use single class import"`
and
`Set Class count to use import with '*' & Names count to use static import with '*' to > 100`